code-challenges.js

```diff
@@ -17,33 +17,115 @@ const fruit4 = "kiwi"
 
 // --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
-const temp1 = 42
-const temp2 = 350
-const temp3 = 212
+// declare variable const and call it temperatureInFahrenheit
+// temperatureInFahrenheit will take any number 
+// use a conditional statement that determines if the number is below boiling point, at boiling point, or above boiling point.
+// the conditional statement is set so that below boiling point is less than 212; boiling point is at exactly 212; above boiling point is anything greater than 212.
+// else is set to run for any string input
+// use test variables provided
+
+//  const temp1 = 42
+//  const temp2 = 350
+//  const temp3 = 212
+
+//  const temperatureInFahrenheit = (temperature) => {
+//     if (temperature === 212){
+//          return`${temperature} is at boiling point`
+// } else if (temperature > 212){
+// return `${temperature} is above boiling point`
+// } else if (temperature < 212) {
+// return `${temperature} is below boiling point`
+// } else {
+// return "enter a number"
+// }
+// }
+// console.log(temperatureInFahrenheit(212))
+
+
 
 
 // --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
 
+// create a new variable declaration combinedNumbers and use built in method .concat to combine myNumbers1 and myNumbers2.
+// use built-in method .length to get the full length of the newly assigned variable combinedNumbers containing the combination of myNumbers1 and myNumbers2 resulting to an output of 10 when console logged.
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+const combinedNumbers = myNumbers1.concat(myNumbers2)
+
+// console.log(myNumbers1)
+// console.log(myNumbers2)
+
+// console.log(combinedNumbers.length)
```

Comment:
This will def get you the correct outcome and is not wrong, but you could consider chaining the `.length` property to the end of line 55.